wireguard/wireguard_freebsd.go

```diff
@@ -1,98 +1,157 @@
 package wireguard
 
 import (
+	"bufio"
+	"fmt"
 	"os"
+	"os/exec"
+	"strconv"
+	"strings"
 
 	"github.com/gravitl/netclient/config"
+	"github.com/gravitl/netclient/ncutils"
 	"github.com/gravitl/netmaker/logger"
-	"github.com/vishvananda/netlink"
+	"golang.zx2c4.com/wireguard/wgctrl/wgtypes"
 )
 
-const kernelModule = "/boot/mdodules/if_wg.ko"
+const kernelModule = "/boot/modules/if_wg.ko"
 
 // Create - creates a linux WG interface based on a node's given config
 func (nc *NCIface) Create() error {
-	if _, err := os.Stat(kernelModule); err == nil {
-		return ApplyWGQuickConf(nc)
+	if _, err := os.Stat(kernelModule); err != nil {
+		logger.Log(3, "using userspace wireguard")
+		return nc.createUserSpaceWG()
```

Comment:
in future need to check if tun module installed for userspace build to function

Reply:
added to GRA-749